<?xml version="1.0"?>
<!--***********************************************************
	Val / CLR Specific Knowledge
	Emily Sheetz, NSTGRO VTE 2024

	run either:
		$ roslaunch safety_aware_reasoning val_clr_specific_knowledge.launch
		OR
		$ rosrun safety_aware_reasoning val_clr_specific_knowledge.py
************************************************************-->
<launch>
	<arg name="robot" default="val"/>
	<arg name="env" default="lunar_habitat"/>
	<arg name="num_points" default="10"/>
	<arg name="max_conds" default="-1"/>

	<node pkg="safety_aware_reasoning" type="val_clr_specific_knowledge.py" name="ValCLRSpecificKnowledgeServerNode" output="screen">
		<param name="robot" type="str" value="$(arg robot)"/>
		<param name="environment" type="str" value="$(arg env)"/>
	</node>
</launch>
